src/main/java/cloud/eppo/logging/BanditAssignment.java

```diff
@@ -0,0 +1,101 @@
+package cloud.eppo.logging;
+
+import cloud.eppo.ufc.dto.Attributes;
+import java.util.Date;
+import java.util.Map;
+
+public class BanditAssignment {
```

Comment:
Would you consider _public final_ fields instead of _getters_? I'm not sure what the java wisdom would say here.

Reply:
My impression is that Java has a weird obsession with getters 😅  The way you access fields for `Record`s seems to confirm this.

Avoiding directly referencing fields will also make interacting with this class consistent with all the others, which I like.